test-pr-builds.sh

```
@@ -0,0 +1,223 @@
+# Function to switch JDK version
+switch_jdk() {
+    local version=$1
+    echo "Switching to JDK $version"
+    sdk use java "$version"
+}
+
+# Function to test PR
+test_pr() {
+    local pr_number=$1
+    local repo=$2
+    local branch=$3
```

Shellcheck (0.10.0)

[warning] branch appears unused. Verify use (or export if used externally).

(SC2034)